editor/editor_inspector.cpp

```diff
@@ -1555,14 +1555,19 @@ void EditorInspector::update_tree() {
 	//to update properly if all is refreshed
 	StringName current_selected = property_selected;
 	int current_focusable = -1;
+	EditorProperty *current_property = nullptr;
 
-	if (property_focusable != -1) {
+	if (object && (property_focusable != -1)) {
```

Comment:
When `object` is null, the method exits early right after clearing the tree, so all of the variables set in this block are not necessary in that case.